tests/pytest/in_person/test_mixins.py

```diff
@@ -6,6 +6,7 @@
 
 
 class TestView(mixins.CommonContextMixin, View):
```

Comment:
The purpose of this view is to test the `CommonContentMixin`? Maybe we just rename it to something like `SampleView` or something so we can avoid the funky dunder attribute?